Podcast/DiscoverViewController.swift

```diff
@@ -77,6 +79,22 @@ class DiscoverViewController: UIViewController, UITableViewDelegate, UITableView
         view.addSubview(categoriesButton)
         
         adjustForScreenSize()
+        
+        
+        let series = Series()
+        series.title = "Planet Money"
+        
+        //episode static data
+        for i in 0..<4 {
+            let episode = Episode(id: i)
+            episode.descriptionText = "44 min • Warriors star Stephen Curry admits he’s getting annoyed by the stream of recent criticism, the possibility ... JHFJSHFGSL igsad asodhaisuhda asidgisag as hsiadgipasug siugdig asuigsi asigasidg asiugdiasgais asigdaisgd aisdgapisdg asidgaosig"
+            episode.smallArtworkImage = #imageLiteral(resourceName: "fillerImage")
+            episode.largeArtworkImage = #imageLiteral(resourceName: "fillerImage")
+            episode.series = series
+            episode.title = "Stephen Curry - EP10"
+            episode.dateCreated = Date.init()
```

Comment:
Probably should create an initializer with more arguments (can use some nice defaults as well). Most of these properties probably can and should become immutable.